### PR TITLE
add wins app

### PR DESCRIPTION
**Pull request by EggsBlue**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by EggsBlue** [comment]
oh yeah. 

all passed! 🎉

File: Casks/w/wins.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+cask "wins" do
+  version "1.6.6"
+  sha256 "594fb383c26f3b36dfa132312ddbafb0e59f7566616b5f02294e769cecf55605"
+
+  url "https://f005.backblazeb2.com/file/winsWebsite/Wins-latest-#{version}.dmg", verified: "backblazeb2.com/file/winsWebsite/"
```
Comment (krehel) [inline Casks/w/wins.rb:5]: `verified` on its own line, indented, and should contain base components of the `url`
```suggestion
  url "https://f005.backblazeb2.com/file/winsWebsite/Wins-latest-#{version}.dmg",
      verified: "f005.backblazeb2.com/file/winsWebsite/"
```

File: Casks/w/wins.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  name "Wins"
+  desc "Window manager"
+  homepage "https://wins.cool/"
+
+  livecheck do
+    url "https://wins.cool/"
+    strategy :page_match
+    regex(/href=.*?Wins-latest[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```
Comment (krehel) [inline Casks/w/wins.rb:13]:
```suggestion
    regex(/href=.*?Wins[._-]latest[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```

**Comment by EggsBlue** [comment]
Thanks

File: Casks/w/wins.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+  url "https://f005.backblazeb2.com/file/winsWebsite/Wins-latest-#{version}.dmg",
+      verified: "f005.backblazeb2.com/file/winsWebsite/"
+  name "Wins"
+  desc "Window manager"
+  homepage "https://wins.cool/"
+
+  livecheck do
+    url "https://wins.cool/"
+    strategy :page_match
```
Comment (miccal) [inline Casks/w/wins.rb:13]:
```suggestion

```
This can be removed, as it is the default strategy.

**Review by miccal — APPROVED** [review]

**Review by krehel — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @EggsBlue.

**Pull request merged**